src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__day calendar__day--day-1"></div>
+      <div class="calendar__day calendar__day--day-2"></div>
+      <div class="calendar__day calendar__day--day-3"></div>
+      <div class="calendar__day calendar__day--day-4"></div>
+      <div class="calendar__day calendar__day--day-5"></div>
+      <div class="calendar__day calendar__day--day-6"></div>
+      <div class="calendar__day calendar__day--day-7"></div>
+      <div class="calendar__day calendar__day--day-8"></div>
```

Comment:
you dont need to hardcode day-classes by yourself, leave it to sass ;)

Reply:
agree, fix @iPanda177 comment
use nth-child instead of creating separate class for each day